contrib/cirrus/setup_environment.sh

```diff
@@ -39,6 +39,15 @@ done
 cd "${GOSRC}/"
 case "${OS_RELEASE_ID}" in
     ubuntu)
+	    apt-get update 
+	    if [[ "$OS_RELEASE_VER" == "19" ]]; then
+		    apt-get purge -y --auto-remove golang* 
```

Comment:
Could I recommend quoting the asterisk?
```bash
   ... --auto-remove "golang*"
```
Without the quotes, if there are one or more files named `golang<something>` in the current directory, the shell will expand that command line to `--auto-remove golang-file1 golang-file2 etc`. I realize this is unlikely and I'm probably being pedantic.